pkg/proxy/iptables/proxier.go

```diff
@@ -591,6 +593,58 @@ func (proxier *Proxier) OnEndpointSlicesSynced() {
 	proxier.syncProxyRules()
 }
 
+// OnNodeAdd is called whenever creation of new node object
+// is observed.
+func (proxier *Proxier) OnNodeAdd(node *v1.Node) {
+	if node.Name != proxier.hostname {
+		klog.Errorf("Received a watch event for a node %s that doesn't match the current node %v", node.Name, proxier.hostname)
+		return
+	}
+	new := node.Labels
```

Comment:
shouldn't use `new`, since it's a go keyword, can we rename to `newLabels`

Reply:
Yes, will fix it.

Reply:
thanks